Formula/liquibase.rb

```diff
@@ -6,6 +6,11 @@ class Liquibase < Formula
   license "Apache-2.0"
   revision 1
 
+  livecheck do
```

Comment:
Maybe we can use https://liquibase.org/download instead?

Reply:
The link from [liquibase.org/download](https://liquibase.org/download) is too long, and not easy for version bump, and I also suspect that it will not work on other times or other computers. It like this:

https://objects.githubusercontent.com/github-production-release-asset-2e65be/2019791/40cb0215-4d8c-4833-b9af-9a4365275a22?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20220316%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20220316T021729Z&X-Amz-Expires=300&X-Amz-Signature=839c398e871530c33e2f92d2ec04ff309e945d75c5aa1474721519fdc5376594&X-Amz-SignedHeaders=host&actor_id=1070090&key_id=0&repo_id=2019791&response-content-disposition=attachment%3B%20filename%3Dliquibase-4.8.0.tar.gz&response-content-type=application%2Foctet-stream

Reply:
We don't need to get the URL from there though, just the version number. And that's in the page itself.

Reply:
Oh, I see, you mean livecheck use [liquibase.org/download](https://liquibase.org/download).